test/sortitionTreeTest.js

```diff
@@ -19,86 +19,122 @@ describe("SortitionTree", () => {
+        // The full output here looks like
+        // 0x00000000,00000000,00000000,00000000,00000000,00000000,00000011,00001234
+        //  slot 7     slot 6   slot 5   slot 4   slot 3   slot 2   slot 1   slot 0
+        // without the commas added for readability. All the padding zeros are
+        // dropped when we hexlify, which simplifies to 0x1100001234.
       })
     })
   })
 
   describe("removeLeaf", () => {
     context("when leaf is removed", () => {
-      beforeEach(async () => {
+      it("should return correct value for the tree", async () => {
         const weight1 = 0x1234
-        const position1 = parseInt("00123456", 8)
+        const position1 = 42798
+
         const weight2 = 0x11
-        const position2 = parseInt("01234567", 8)
+        const position2 = 342391
 
         const leaf1 = await sortition.toLeaf(alice.address, weight1)
         await sortition.publicSetLeaf(position1, leaf1, weight1)
 
         const leaf2 = await sortition.toLeaf(bob.address, weight2)
         await sortition.publicSetLeaf(position2, leaf2, weight2)
         await sortition.publicRemoveLeaf(position1)
-      })
-
-      it("should return correct value for the tree", async () => {
         const root = await sortition.getRoot()
         expect(ethers.utils.hexlify(root)).to.be.equal("0x1100000000")
+        // The full output here looks like
+        // 0x00000000,00000000,00000000,00000000,00000000,00000000,00000011,00000000
+        //  slot 7     slot 6   slot 5   slot 4   slot 3   slot 2   slot 1   slot 0
+        // without the commas added for readability. All the padding zeros are
+        // dropped when we hexlify, which simplifies to 0x1100000000.
       })
     })
   })
 
   describe("insertOperator", () => {
     const weightA = 0xfff0
     const weightB = 0x10000001
+    // weightA + weightB = 0x1000fff1
 
     context("when operators are inserted", () => {
-      beforeEach(async () => {
+      it("should return correct value for the tree", async () => {
+        // insertion begins left to right, so alice is inserted at position 0,
+        // and bob is inserted at position 1. Their weights will propagate to
+        // the root's first slot.
         await sortition.publicInsertOperator(alice.address, weightA)
         await sortition.publicInsertOperator(bob.address, weightB)
-      })
-
-      it("should return correct value for the tree", async () => {
         const root = await sortition.getRoot()
-        expect(ethers.utils.hexlify(root)).to.be.equal("0x1000fff1")
+        expect(ethers.utils.hexlify(root)).to.be.equal("0x1000fff1") // weightA + weightB
+        // The full output here looks like
+        // 0x00000000,00000000,00000000,00000000,00000000,00000000,00000000,1000fff1
+        //  slot 7     slot 6   slot 5   slot 4   slot 3   slot 2   slot 1   slot 0
+        // without the commas added for readability. All the padding zeros are
+        // dropped when we hexlify, which simplifies to 0x1100000000.
```

Comment:
It simplifies to `0x1000fff1`

Reply:
right you are sir

Reply:
[6241b8e](https://github.com/keep-network/sortition-pools/pull/180/commits/6241b8e5fbf9d1878c92ae2b71c60a2393d736d8)